lib/psulib_traject/processors/call_number/dewey.rb

```diff
@@ -15,7 +47,32 @@ def scheme
     end
 
     def reduce
-      call_number
+      value = remove_by_regex
+      value = value[0...(value.index(LOOSE_MONTHS_REGEX) || value.length)] # remove loose months
+
+      if serial
+        self.class.remove_years(value)
+      else
+        value.strip
+      end
     end
+
+    private
+
+      # @note These are the original regex patterns from Stanford.
+      def remove_by_regex
+        case removables
+        when VOL_PATTERN
+          call_number.slice(0...call_number.index(removables[VOL_PATTERN])).strip
+        when VOL_PATTERN_LOOSER
+          call_number.slice(0...call_number.index(removables[VOL_PATTERN_LOOSER])).strip
+        when VOL_PATTERN_LETTERS
+          call_number.slice(0...call_number.index(removables[VOL_PATTERN_LETTERS])).strip
+        when ADDL_VOL_PATTERN
+          call_number.slice(0...call_number.index(removables[ADDL_VOL_PATTERN])).strip
+        else
+          call_number
+        end
+      end
```

Comment:
can all of this just be moved into `Processors::CallNumber::Base`?

Reply:
hmm might be able to. will try that

Reply:
hmm, it uses `removeables` which does not exist in the base class 🤔

Reply:
It's just spelled differently in Dewey as "removables" instead of "removeables". Either way is correct, AFAICT. But, the regexes in the init methods are _very_ slightly different.

I pushed up this branch to show what a combined setup might look like: https://github.com/psu-libraries/psulib_traject/tree/318-dewey-awead

Reply:
@awead thanks for pushing the changes up. got them 👍🏻